llama-index-integrations/multi_modal_llms/llama-index-multi-modal-llms-reka/pyproject.toml

```diff
@@ -0,0 +1,51 @@
+[build-system]
+build-backend = "poetry.core.masonry.api"
+requires = ["poetry-core"]
+
+[tool.codespell]
+check-filenames = true
+check-hidden = true
+# Feel free to un-skip examples, and experimental, you will just need to
+# work through many typos (--write-changes and --interactive will help)
+skip = "*.csv,*.html,*.json,*.jsonl,*.pdf,*.txt,*.ipynb"
+
```

Comment:
lets add the tool.llamahub and tool.llamahub.class_authors sections here as well